sycl/test/esimd/ctor_codegen.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clangxx -Xclang -no-opaque-pointers -fsycl -fsycl-device-only -S %s -o - | FileCheck %s
+// RUN: %clangxx -Xclang -opaque-pointers -fsycl -fsycl-device-only -S %s -o - | FileCheck %s
```

Comment:
```suggestion
// RUN: %clangxx -fsycl -fsycl-device-only -S %s -o - | FileCheck %s
```

Let's remove uses of `-opaque-pointers` flag from the tests. llorg has removed this flag and we need to do so as well. This flag is used by default, so setting it has no benefits and just require additional change in the near future.

Reply:
done in latest commit, thx

Reply:
Great! Thanks a lot! Sorry for reviewing unfinished PR.

Reply:
np, faster this way anyway :)